test/Locker.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Contract } from "ethers";
+import { expect } from "./chai-setup";
+import { pack, keccak256 } from "@ethersproject/solidity";
+import { ethers } from "hardhat";
+import {
+  setupNftGemGovernor
+} from './fixtures/Governance.fixture';
+
+describe('Locker Test Suite', function () {
+  let locker: Contract;
+  let NFTGemMultiToken: any;
+  const tokenHash = keccak256(['bytes'], [pack(['string'], ['Test Token'])]);
```

Comment:
No need to make a hash. you can just use '1' , '2' etc for different token types